.circleci/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This script setup an working environemnt for running glow tests and gtest driver.
```

Comment:
can you add that we run with the enabled CPU backend and disabled OpenCL backend.

Reply:
Sure